css/normalize.css

```diff
@@ -1,3 +1,4 @@
+/*david*/
```

Comment:
remove all comments - both auto-generated and explanations - no need to explain simple lines like `margin: 0;`